internal/anchor/reconciler.go

```diff
@@ -424,7 +424,7 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 	}
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&api.SubnamespaceAnchor{}).
-		Watches(&source.Channel{Source: r.affected}, &handler.EnqueueRequestForObject{}).
-		Watches(&source.Kind{Type: &corev1.Namespace{}}, handler.EnqueueRequestsFromMapFunc(nsMapFn)).
+		WatchesRawSource(&source.Channel{Source: r.affected}, &handler.EnqueueRequestForObject{}).
```

Comment:
I know one is supposed to use Watches(), but I don't understand the builder stuff and how to update these callers for the new type-safe API.

Reply:
`WatchesRawSource` seems ok (for now at least). Refering again to the release notes its the prefered way to use `source` and since we aren't using predicates I think we should be fine here:

> Improve builder package capabilities and general UX ([#2135](https://github.com/kubernetes-sigs/controller-runtime/pull/2135))
> 
>     builder.Builder.Watches signature has changed, it now takes a client.Object instead of a source.Source as first argument.
>         For source.Source, use WatchesRawSource.